src/fungible_token.rs

```diff
@@ -398,6 +400,7 @@ impl FungibleToken {
         self.internal_storage_balance_of(account_id).unwrap()
     }
 
+    #[allow(dead_code)]
```

Comment:
This is the new annotation I mentioned in the PR description. I don't know what `storage_unregister` is for. cc @mrLSD